src/OpenTelemetry.Api/Logs/LoggerProvider.cs

```diff
@@ -6,28 +6,16 @@
 #if NETSTANDARD2_1_OR_GREATER || NET6_0_OR_GREATER
 using System.Diagnostics.CodeAnalysis;
 #endif
-#if NET8_0_OR_GREATER
+#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
 using OpenTelemetry.Internal;
 #endif
 
 namespace OpenTelemetry.Logs;
 
-#if EXPOSE_EXPERIMENTAL_FEATURES
 /// <summary>
 /// LoggerProvider is the entry point of the OpenTelemetry API. It provides access to <see cref="Logger"/>.
 /// </summary>
-/// <remarks><inheritdoc cref="Logger" path="/remarks"/></remarks>
-#if NET8_0_OR_GREATER
-[Experimental(DiagnosticDefinitions.LoggerProviderExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
-#endif
-public
-#else
-/// <summary>
-/// LoggerProvider is the entry point of the OpenTelemetry API. It provides access to <see cref="Logger"/>.
-/// </summary>
-internal
-#endif
-    class LoggerProvider : BaseProvider
+public class LoggerProvider : BaseProvider
```

Comment:
Is this needed outside of hosting package?

Reply:
The main thing exposing this does is let users call `ForceFlush` if they need to. They can also call `Shutdown` or `AddProcessor` if needed. None of those scenarios are possible today so it is mostly a spec compliance thing.

How would you do that? Something like this...

```csharp
builder.Services.AddOpenTelemetry()
    .WithLogging(logging => logging.AddOtlpExporter());

var app = builder.Build();

var loggerProvider = app.Services.GetRequiredService<LoggerProvider>();

loggerProvider.ForceFlush();
```

PS: The Log Bridge API is all rooted off `LoggerProvider` too but that isn't exposed yet.